airflow/providers/presto/hooks/presto.py

```diff
@@ -112,8 +112,20 @@ def get_isolation_level(self) -> Any:
     def _strip_sql(sql: str) -> str:
         return sql.strip().rstrip(';')
 
+    @overload
+    def get_records(self, sql: str = "", parameters: Optional[dict] = None):
+        """Get a set of records from Presto
+        :param sql: SQL statement to be executed.
+        :param parameters: The parameters to render the SQL query with.
+        """
+        ...
```

Comment:
```suggestion
        """Get a set of records from Presto.

        :param sql: SQL statement to be executed.
        :param parameters: The parameters to render the SQL query with.
        """
```

One blank line after the first line in the docstring. (See [PEP 257](https://www.python.org/dev/peps/pep-0257/))

The `...` part is only needed so the function is not empty, but we already have a docstring here and don’t need it.

Reply:
Made changes to docstring and also removed `...`.